book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'rental'
+
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] According to project requirement, there should be a method `add_rental(person, date)` which will `create the new rental achieving the 'has-many' association` kindly fix this